admin-dev/themes/new-theme/js/pages/product/edit/index.js

```diff
@@ -46,6 +48,9 @@ $(() => {
 
   const $productForm = $(ProductMap.productForm);
 
+  // Init product model along wth input watching and syncing
+  new ProductModel($productForm, window.prestashop.instance.eventEmitter);
```

Comment:
Wouldn't be the responsability of the ProductModel to select the form ? As it looks like the `$productForm` var is not used anywhere else

Reply:
Yeah I can do that indeed

Reply:
Mmm, actually the form is used to get the productId from data attribute which is useful to detect if we're in edition mode